app/apps/users/views.py

```diff
@@ -17,3 +26,31 @@ def get_queryset(self):
         else:
             user = self.request.user
             return User.objects.filter(creator=user)
+
+    def create(self, request, *args, quantity, **kwargs):
+        r = requests.get(f"https://randomuser.me/api/?results={quantity}")
+        generated_data = []
+        
+        if r.status_code == 200:
+            d = json.dumps(r.json())
+            data = json.loads(d)
+            
+            for i in range(0, int(len(data))):
+                credentials = User(creator=self.request.user,
+                                   gender=data['results'][0]['gender'],
+                                   first_name=data['results'][0]['name']['first'],
+                                   last_name=data['results'][0]['name']['last'],
+                                   country=data['results'][0]['location']['country'],
+                                   city=data['results'][0]['location']['city'],
+                                   email=data['results'][0]['email'],
+                                   username=data['results'][0]['login']['username'],
+                                   phone=data['results'][0]['phone'],
+                                   picture=data['results'][0]['picture']['medium'],
+                                   )
+
+                credentials.save()
```

Comment:
Google it out, but much more better would be to use one `bulk_create` vs multiple `save`